event-based-simulators/main/event_based_simulators.py

```diff
@@ -465,16 +458,21 @@ def load(filename):
 SHIFTPLANS_MODELS = load("shiftplans.json")
 shiftplans = list(map(lambda model: Shiftplan(model), SHIFTPLANS_MODELS))
 
-if CREATE_PROFILES.lower() == "true":
-    [oeeAPI.create_and_activate_profile(id, ProfileCreateMode.CREATE_IF_NOT_EXISTS) 
+if DELETE_PROFILES.lower() == "true":
+    log.debug(f'Deleting all Profiles')
+    oeeAPI.delete_all_simulators_profiles()
+
+if PROFILE_CREATE_MODE:    
```

Comment:
I'm not sure I understand the syntax; does that mean that the following block is only executed if the variable 'PROFILE_CREATE_MODE' is set? But is it even possible that it is not set? b/c we have a default value defined in line 17

Reply:
True. This is still the syntax that i used when PROFILE_CREATE_MODE was a boolean